scripts/hal_alignment/hal_gene_liftover.py

```diff
@@ -154,7 +163,11 @@ def parse_region(region: str) -> SimpleRegion:
             with Popen(cmd2, stdin=p1.stdout) as p2:
                 p2.wait()
                 if p2.returncode != 0:
-                    raise RuntimeError(f'pslPosTarget terminated with signal {-p2.returncode}')
+                    status_type = 'exit code' if p2.returncode > 0 else 'signal'
+                    raise RuntimeError(
+                        f'pslPosTarget terminated with {status_type} {abs(p2.returncode)}')
```

Comment:
If this gets any more complex (and the same applied below), I would encourage to do a "returncode function" to process it (just for future reference).

Reply:
Fair enough, it is starting to get quite complicated.